### DISCUSSION TITLE
REST API example with `in` filter?

### DISCUSSION THREAD

**Discussion by raevilman** [supabase/supabase#1069] [Questions] [answered]
As explained on [this page](https://supabase.io/docs/reference/javascript/in) for js sdk, what would the equivalent for REST API?

Actually, I want to send list of IDs to the `DELETE` endpoint.

I tried following:
```
curl -X DELETE 'https://example.supabase.co/rest/v1/dummy_items?some_column=in.[266793,266789]' \
-H "apikey: SUPABASE_KEY" \
-H "Authorization: Bearer SUPABASE_KEY"
```

But got error:
```
Response: {"details": "unexpected \"[\" expecting \"(\"", "message": "\"failed to parse filter (in.[26679392,26678970])\" (line 1, column 4)"}
```

Let me know if something like this is possible in one call, other wise I've to call `DELETE` endpoint in loop!!

**Comment by silentworks** [comment, marked as answer]
This should work but I think you need to encode the open and closing array brackets and add the values in quotation marks or use the quotation mark encoding instead `%22`.

```sh
curl -X DELETE 'https://example.supabase.co/rest/v1/dummy_items?some_column=in.%28"266793","266789"%29' \
-H "apikey: SUPABASE_KEY" \
-H "Authorization: Bearer SUPABASE_KEY"
```

**Reply by raevilman** [reply]
Awesome 🎉🥳. 

It did work. 
Just a small typo there, its `%29` for the closing brackets.

Thank you @silentworks 👏

**Reply by silentworks** [reply]
Thanks fixed.

**Reply by TechBoyy6** [reply]
i want similar but for reading the value, can someone help?

**Reply by steve-chavez** [reply]
Use GET instead of DELETE

**Reply by hcsoft19** [reply]
This should also work
```sh
curl --location 'https://example.supabase.co/rest/v1/dummy_items?some_column=in.(266793%2C266789)' \
-H "apikey: SUPABASE_KEY" \
-H "Authorization: Bearer SUPABASE_KEY"
```
